js/blocks/ProgramBlocks.js

```diff
@@ -95,13 +137,39 @@ function setupProgramBlocks(activity) {
 
             this.formBlock({
                 //.TRANS: save the heap contents to a URL
```

Comment:
The trans note has to stay just above the line with the string.
                /**
                 * The name of the block.
                 * @type {string}
                 */
                //.TRANS: save the heap contents to a URL
                name: _("save heap to App"),